From aa9e403f48d6c21bd2125c1f87aa53a35f379c81 Mon Sep 17 00:00:00 2001
From: Robbe Gaeremynck <robbe.gaeremynck@outlook.com>
Date: Mon, 8 Jun 2026 15:28:56 +0200
Subject: [PATCH] Add arguments to function declarations in psu_init_gpl.c

---
 psu_init_gpl.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/psu_init_gpl.c b/psu_init_gpl.c
index 9b7f39f..34a4bc0 100644
--- a/psu_init_gpl.c
+++ b/psu_init_gpl.c
@@ -35,7 +35,7 @@ static void mask_delay(u32 delay);
 
 static u32 mask_read(u32 add, u32 mask);
 
-static int serdes_rst_seq (u32 lane3_protocol, u32 lane3_rate, u32 lane2_protocol, u32 lane2_rate, u32 lane1_protocol, u32 lane1_rate, u32 lane0_protocol, u32 lane0_rate);
+static int serdes_rst_seq (u32 pllsel, u32 lane3_protocol, u32 lane3_rate, u32 lane2_protocol, u32 lane2_rate, u32 lane1_protocol, u32 lane1_rate, u32 lane0_protocol, u32 lane0_rate);
 
 static int serdes_bist_static_settings(u32 lane_active); 
 
@@ -43,7 +43,7 @@ static int serdes_bist_run(u32 lane_active);
 
 static int serdes_bist_result(u32 lane_active);
 
-static int serdes_illcalib_pcie_gen1 (u32 lane3_protocol, u32 lane3_rate, u32 lane2_protocol, u32 lane2_rate, u32 lane1_protocol, u32 lane1_rate, u32 lane0_protocol, u32 lane0_rate, u32 gen2_calib);
+static int serdes_illcalib_pcie_gen1 (u32 pllsel, u32 lane3_protocol, u32 lane3_rate, u32 lane2_protocol, u32 lane2_rate, u32 lane1_protocol, u32 lane1_rate, u32 lane0_protocol, u32 lane0_rate, u32 gen2_calib);
 
 static int serdes_illcalib (u32 lane3_protocol, u32 lane3_rate, u32 lane2_protocol, u32 lane2_rate, u32 lane1_protocol, u32 lane1_rate, u32 lane0_protocol, u32 lane0_rate);
 
-- 
2.43.0

